shared/constants/chat.js

```diff
@@ -778,6 +798,11 @@ function messageKeyKindIsMessageID(key: MessageKey): boolean {
 }
 
 function messageKeyKind(key: MessageKey): MessageKeyKind {
+  try {
```

Comment:
What's going on here? Debug code you forgot to remove?

Reply:
Oops, yes, sorry about that!